src/main/java/ua/lviv/iot/parkingServer/logic/services/interfaces/ParkingSpotServiceInterface.kt

```diff
@@ -0,0 +1,6 @@
+package ua.lviv.iot.parkingServer.logic.services.interfaces
+
+import ua.lviv.iot.parkingServer.model.ParkingSpot
+
+interface ParkingSpotServiceInterface : GeneralServiceInterface<ParkingSpot, Long> {
+}
```

Comment:
Remove brackets because your interface has empty body.
Add newline at the end of file